external-api/src/v1/controllers/estore/eStore.controller.ts

```diff
@@ -150,123 +150,130 @@ export const create = async (req: EstoreRequest, res: Response): Promise<Respons
+    }
 
-      // Add CRON job to the collection
-      const inserted = await EstoreRepo.insertOne({
-        payload: eStoreData,
-        timestamp: getNowAsEpoch(),
-        cron: {
-          site: { status: ESTORE_CRON_STATUS.PENDING },
-          term: { status: ESTORE_CRON_STATUS.PENDING },
-          buyer: { status: ESTORE_CRON_STATUS.PENDING },
-          deal: { status: ESTORE_CRON_STATUS.PENDING },
-          facility: { status: ESTORE_CRON_STATUS.PENDING },
-          document: { status: ESTORE_CRON_STATUS.PENDING },
-        },
-      });
+    /**
+     * Send `201` status code back to avoid
+     * `TFM-API` awaiting.
+     */
+    console.info('Attempting to create a new CRON job for deal %s', dealIdentifier);
 
-      if (!inserted) {
-        throw new Error('eStore CRON job log insertion failed');
-      }
+    // Add CRON job to the collection
+    const inserted = await EstoreRepo.insertOne({
+      payload: eStoreData,
+      timestamp: getNowAsEpoch(),
+      cron: {
+        site: { status: ESTORE_CRON_STATUS.PENDING },
+        term: { status: ESTORE_CRON_STATUS.PENDING },
+        buyer: { status: ESTORE_CRON_STATUS.PENDING },
+        deal: { status: ESTORE_CRON_STATUS.PENDING },
+        facility: { status: ESTORE_CRON_STATUS.PENDING },
+        document: { status: ESTORE_CRON_STATUS.PENDING },
+      },
+    });
 
-      const { insertedId: _id } = inserted as InsertOneResult;
+    if (!inserted) {
+      throw new Error('eStore CRON job log insertion failed');
+    }
 
-      // Site exists check
-      console.info('Initiating eStore site existence check for exporter %s', exporterName);
-      const siteExistsResponse: SiteExistsResponse | EstoreErrorResponse = await siteExists(exporterName);
+    const { insertedId: _id } = inserted as InsertOneResult;
 
-      const created = siteExistsResponse?.data?.status === ESTORE_SITE_STATUS.CREATED;
-      const provisioning = siteExistsResponse?.data?.status === ESTORE_SITE_STATUS.PROVISIONING;
-      const absent = siteExistsResponse?.data?.status === ESTORE_SITE_STATUS.NOT_FOUND;
+    // Site exists check
+    console.info('Initiating eStore site existence check for exporter %s', exporterName);
+    const siteExistsResponse: SiteExistsResponse | EstoreErrorResponse = await siteExists(exporterName);
 
-      // Site already exists in eStore
-      if (created) {
-        /**
-         * Update record-set with the site name.
-         * Update `cron-job-logs`
-         */
-        await EstoreRepo.updateByDealId(dealId, {
-          'cron.site.create': {
-            status: ESTORE_CRON_STATUS.COMPLETED,
-            response: siteExistsResponse.data.status,
-            timestamp: getNowAsEpoch(),
-            id: siteExistsResponse.data.siteId,
-          },
-          'cron.site.status': ESTORE_CRON_STATUS.COMPLETED,
-        });
+    const created = siteExistsResponse?.data?.status === ESTORE_SITE_STATUS.CREATED;
```

Comment:
could this be more descriptive? siteCreated?

Reply:
Good shout, have updated the variable names.